.github/workflows/test.yml

```diff
@@ -9,6 +9,17 @@ jobs:
     timeout-minutes: 5
     runs-on: ubuntu-latest
     steps:
+      - name: Test redis cluster
+        uses: vishnudxb/redis-cluster@1.0.9
```

Comment:
Is it ok to use some external action like this?

Reply:
Does this mean these tests won't run at all without redis locally either? @jan-auer wondering how we might mark acceptance tests like this different from our typical unit tests.

Reply:
@asottile-sentry We usually pin versions by hash right?

Reply:
> Does this mean these tests won't run at all without redis locally either? @jan-auer wondering how we might mark acceptance tests like this different from our typical unit tests.

Yeah we'll require redis, you can run it using devservices. For now that seems simpler then making our own devservices

Reply:
Yeah agree. Just wondering if there should be a way to mark these as like "acceptance" tests

Reply:
yeah it would be best to pin this action to a hash which is good enough to prevent drift

we'll hopefully have some stuff *soon* which makes this easier (without needing to use `sentry devservices ...`) and will allow any service at sentry to have reusable dependent services without needing `sentry` at all (basically `devservices` but independent and better!)